### PR TITLE
hacking Pun added!

### PR DESCRIPTION
**Pull request by NilanshBansal**

### CODE REVIEW

File: puns.json [hunk 1]
```diff
@@ -250,5 +250,13 @@
...
     "pun": "Q: What do you call a program whose file you can't find?",
     "punchline": "A: Soft-where!"
+  },
+  {
+    "pun": "Q: What grade did the programmer get on his test?",
+    "punchline": "A: C++"
+  },
+  {
+    "pun": "Q: What made the technician win the golf tournament?",
+    "punchline": "A: He had a hard-drive!"
```
Comment (AlexLakatos) [inline puns.json]:
```suggestion
    "punchline": "A: They had a hard-drive!"
```

File: puns.json [hunk 2]
```diff
@@ -250,5 +250,13 @@
   {
     "pun": "Q: What do you call a program whose file you can't find?",
     "punchline": "A: Soft-where!"
+  },
+  {
+    "pun": "Q: What grade did the programmer get on his test?",
```
Comment (AlexLakatos) [inline puns.json]:
```suggestion
    "pun": "Q: What grade did the programmer get on their test?",
```

**Comment by AlexLakatos** [comment]
@NilanshBansal thanks for the update. The updates I did didn't refer to "they" as a plural, the English language has a singular form of "they" used as a gender-neutral singular pronoun. 

https://en.wikipedia.org/wiki/Singular_they

**Comment by NilanshBansal** [comment]
> @NilanshBansal thanks for the update. The updates I did didn't refer to "they" as a plural, the English language has a singular form of "they" used as a gender-neutral singular pronoun.
> 
> https://en.wikipedia.org/wiki/Singular_they
> 
> Can you please revert you last commit, and then this should be good to merge. 🙌

Thanks, @AlexLakatos it's always good learning working with you!

**Pull request merged**